index.html

```diff
@@ -94,45 +96,155 @@ <h1>MY RECENT
+                </div>
+                <div class="specs" id="specs4">
+                    <div class="hero1">
+                    <h1>
+                        Multi-Post Stories
+                    </h1>
+                    </div>
+                    <div class="p">
+                    <p>
+                        A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    </div>
+                    <div class="ul">
+                    <ul>
+                        <li id="css4">css</li>
+                        <li id="html4">html</li>
+                        <li id="bootstrap4">bootstrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                    </div>
+                    <div class="submit">
                     <button type="submit" class="btn">
                         See Project
                     </button>
+                    </div>
                 </div>
             </div>
         </section>
         <section class="footer">
             <div class="form">
-              <h1>CONTACT US</h1>
-              <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                <div>
+              <h1>Let's chat</h1>
+            </div>
+            <div>
+              <p id="form-p">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                </div>
+                <div>
               <form action="https://formspree.io/f/xlezyzng" method="post">
                 <input type="text"  name="name" class="name" placeholder="Name" maxlength="30" required>
-                <input type="email" name="email" class="email" placeholder="Email" required>
+                <input type="email" name="email" class="email1" placeholder="Email" required>
                 <textarea name="message" class="text" id="message" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
                 <button type="submit" class="btn">Submit</button>
               </form>
+            </div>    
             </div>
         </section>
-        <br>
-        <br>
+        <hr>
         <div class="icons">   
             <a href="#"><img alt= 'git' src="./assets/Github.svg"></a>
             <a href="#"><img alt= 'Vic' src="./assets/Victory.svg"></a>
             <a href="#"><img alt= 'Twitter' src="./assets/Twitter.svg"></a>
             <a href="#"><img alt= 'M' src="./assets/M.svg"></a>
-        </div>   
+        </div>
+        <div class="footer-email">
+            <h4>email@email.com</h4>
+        </div>           
```

Comment:
- [OPTIONAL] Great job @OmarMWarraich 💯🎉 I think this is the only issue you have. The email after the social media icon links shares the same row as the social media icon links based on your Figma Design in `desktop version`, So I suggest fixing this small issue 👌

Reply:
@Bondok6 
- I deeply appreciate your valued input and assure you that they shall be fixed asap.